tests/unit/store/getters.spec.ts

```diff
@@ -9,6 +9,7 @@ describe( 'getters', () => {
 				conditionRow: {
 					valueData: { value: 'foo' },
 					propertyData: { id: 'P123', label: 'abc' },
+					propertyValueRelationData: { value: 'with' },
```

Comment:
same value = 'matching'

Reply:
once we make an enum for this we might as well do something like: 
`propertyValueRelation: PropertyValueRelation.Matching,
`

Reply:
updated